sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -64,3 +65,17 @@ export interface InstitutionLocationsSummaryDto {
     postalCode: string;
   };
 }
+
+/**
+ * Possible status for a designation location agreement.
+ */
+export enum DesignationLocationAgreementStatus {
```

Comment:
I guess this status enum can be represented for designation approval status of an institution and also location. The enum name DesignationLocationAgreementStatus is giving a feel like it is for location only. 

I would recommend renaming the enum to DesignationApprovalStatus or DesignatedStatus or something event better.